applications/openshift/general/resource_requests_quota/oval/shared.xml

```diff
@@ -0,0 +1,11 @@
+<def-group oval_version="5.11">
+  <definition class="compliance" id="resource_requests_quota" version="1">
+    {{{ oval_metadata("The sysctl parameter needs to be set before enabling kernel protection") }}}
+
+    <criteria operator="OR">
+      <extend_definition comment="cluster quotas enabled" definition_ref="resource_requests_quota_cluster" />
+      <extend_definition comment="resource quota per project" definition_ref="resource_requests_quota_per_project" />
+    </criteria>
```

Comment:
There is something that seems really weird, all the unit cases passed for  `resource_requests_quota` where either `resource_requests_quota_cluster` passes or `resource_requests_quota_per_project` passes, however, that's not the case when the check was running in the cluster, the main rule fails when `resource_requests_quota_cluster` passes, did I write anything wrong maybe in here? @JAORMX